### PR TITLE
add k8s console log show watchdog log path

### PR DESCRIPTION
**Pull request by YanjieGao**
also increase watchdog log file max size

### CODE REVIEW

File: prometheus/exporter/watchdog.py [hunk 1]
```diff
@@ -321,14 +323,26 @@ def main(argv):
...
+
+    loggerRoot.setLevel(logging.INFO)  
+    loggerRoot.propagate = False
+    ch = logging.StreamHandler()
+    ch.setLevel(logging.INFO)
+    # create formatter and add it to the handlers
+    formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+    ch.setFormatter(formatter)
+    loggerRoot.addHandler(ch) 
+    loggerRoot.info("Please view log trace:")
```
Comment (fanyangCS) [inline prometheus/exporter/watchdog.py]: >loggerRoot.info("Please view log trace:")  [](start = 4, length = 42)

remove

File: prometheus/exporter/watchdog.py [hunk 2]
```diff
@@ -321,14 +323,26 @@ def main(argv):
...
+    loggerRoot.propagate = False
+    ch = logging.StreamHandler()
+    ch.setLevel(logging.INFO)
+    # create formatter and add it to the handlers
+    formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+    ch.setFormatter(formatter)
+    loggerRoot.addHandler(ch) 
+    loggerRoot.info("Please view log trace:") 
+    loggerRoot.info("Log trace path {}/watchdog.log".format(logDir))
+
```
Comment (fanyangCS) [inline prometheus/exporter/watchdog.py]: in loggerRoot, please explain why you do not out log in k8s log directly. 
also, please output the current configuration of logger (e.g.,, maxBytes, etc.)

File: prometheus/exporter/watchdog.py [hunk 3]
```diff
@@ -25,7 +25,9 @@
 import time
 import common
 
+loggerRoot = logging.getLogger()  
 logger = logging.getLogger("watchdog")  
+loggerCommon = logging.getLogger("common")
```
Comment (sterowang) [inline prometheus/exporter/watchdog.py]: Console displays "No handlers could be found for logger "common"", why we need to deal with "common"?

Comment (YanjieGao) [reply prometheus/exporter/watchdog.py]: use pai/pai-management/k8sPaiLibrary/maintainlib/common.py   ssh method. common.py also call logging lib and will generate "No handlers could be found for logger "common""

Comment (YanjieGao) [reply prometheus/exporter/watchdog.py]: If not deal with it, it log will also up to root logger and keep generate to console

**Pull request merged**